src/hearings/containers/hearing-actuals/hearing-actuals-timing/hearing-actuals-timing.component.html

```diff
@@ -0,0 +1,80 @@
+<div class="govuk-width-container">
+  <a class="govuk-back-link" [routerLink]="['../hearing-actual-add-edit-summary']">Back</a>
+  <main class="govuk-main-wrapper" id="main-content" role="main">
+    <div class="govuk-grid-row" *ngIf="form">
+      <form [formGroup]="form" (submit)="submit(form.value, form.valid)">
+        <span class="govuk-caption-l">Jane Smith vs DWP</span>
```

Comment:
Can this hard coded case name move to the ts and add the // TODO so we won't miss this if we have known where the case name comes from.

Reply:
@johnbenjamin-hmcts has done something similar in his pr https://github.com/hmcts/rpx-xui-webapp/pull/1785/files